pygit2/remote.py

```diff
@@ -344,6 +344,16 @@ def create(self, name, url, fetch=None):
 
         return Remote(self._repo, cremote[0])
 
+    def create_anonymous(self, url):
+        """Create a new anonymous (in-memory only) remote with the given URL.
+        Returns a <Remote> object.
+        """
```

Comment:
This could also be done in the existing `create()` by checking something like
```
if not name:
    C.git_remote_create_anonymous
```
but I wasn't sure if that would be explicit enough since `create(None, url)` could also imply that the user intended to make a libgit2 `detached` remote instead of an `anonymous` one.